Dockerfile

```diff
@@ -6,12 +6,12 @@ WORKDIR /app/
 
 # Copy project dependencies
 COPY package*.json /app/
-COPY yarn.lock /app/
+COPY package-lock.json /app/
```

Comment:
This line is redundant with the one above, right?

Reply:
good catch